test/test-lib.sh

```diff
@@ -154,6 +154,15 @@ expect_fail()
 	return 1
 }
 
+expect_success()
```

Comment:
All tests run with `set -e`. So implicitly every command is prefixed by `expect_success`.